tests/v2/test_waku_rln_relay_onchain_zerokit.nim

```diff
@@ -0,0 +1,472 @@
+
+# contains rln-relay tests that require interaction with Ganache i.e., onchain tests
+{.used.}
+
+import
+  std/options,
+  testutils/unittests, chronos, chronicles, stint, web3, json,
+  stew/byteutils, stew/shims/net as stewNet,
+  libp2p/crypto/crypto,
+  ../../waku/v2/protocol/waku_rln_relay/rlnzerokit/[waku_rln_relay_utils,   
+      waku_rln_relay_types, rln_relay_contract],                            ##TODO: remove rlnzerokit ref once zerokit rln is merged
+  ../../waku/v2/node/wakunode2,
+  ../test_helpers,
+  ./test_utils
+
+const RLNRELAY_PUBSUB_TOPIC = "waku/2/rlnrelay/proto"
+const RLNRELAY_CONTENT_TOPIC = "waku/2/rlnrelay/proto"
+
+#  contract ABI
+contract(MembershipContract):
+  proc register(pubkey: Uint256) # external payable
+  proc MemberRegistered(pubkey: Uint256, index: Uint256) {.event.}
+  # proc registerBatch(pubkeys: seq[Uint256]) # external payable
```

Comment:
Will these be implemented later?

Reply:
Yes, but right now, there is no actual use case for them.